AlgorithmFactory/Python/Wrappers/AlgorithmPythonWrapper.cs

```diff
@@ -870,5 +868,56 @@ public void SetParameters(Dictionary<string, string> parameters)
         {
             _baseAlgorithm.SetParameters(parameters);
         }
+
+        /// <summary>
+        /// Creates Util module
+        /// </summary>
+        /// <returns>PyObject with utils</returns>
+        private PyObject ImportUtil()
+        {
+            var code =
```

Comment:
This should be put into a separate file; copied to the output directory and loaded as a string.